backdoor/src/main/java/org/codice/acdebugger/backdoor/Backdoor.java

```diff
@@ -465,15 +479,15 @@ private Bundle getBundle0(@Nullable Object obj) {
     return bundle;
   }
 
-  private String getServicePermissionInfoAndGrant(
+  private String getServicePermissionInfoAndGrant0(
       String bundle, ProtectionDomain domain, ServiceEvent serviceEvent, boolean grant)
       throws Exception {
     synchronized (this) {
       final ServiceReference sr = serviceEvent.getServiceReference();
       final ServicePermission p = new ServicePermission(sr, ServicePermission.GET);
       final String[] objectClass = (String[]) sr.getProperty(Constants.OBJECTCLASS);
       final boolean implies = domain.implies(p);
-      final Set<String> implied = new HashSet<>(12);
+      final Set<String> implied = new LinkedHashSet<>(12);
```

Comment:
ℹ️ Although not required to be ordered, it does simplify testing and make tests more predictable